apps/src/applab/applab.js

```diff
@@ -761,90 +739,118 @@ Applab.init = function(config) {
   AppStorage.populateTable(level.dataTables, false); // overwrite = false
   AppStorage.populateKeyValue(level.dataProperties, false); // overwrite = false
 
-  studioApp.init(config);
-
-  var viz = document.getElementById('visualization');
-  var vizCol = document.getElementById('visualizationColumn');
-
-  if (!config.noPadding) {
-    viz.className += " with_padding";
-    vizCol.className += " with_padding";
-  }
-
-  if (config.embed || config.hideSource) {
-    // no responsive styles active in embed or hideSource mode, so set sizes:
-    viz.style.width = Applab.appWidth + 'px';
-    viz.style.height = (shouldRenderFooter() ? Applab.appHeight : Applab.footerlessAppHeight) + 'px';
-    // Use offsetWidth of viz so we can include any possible border width:
-    vizCol.style.maxWidth = viz.offsetWidth + 'px';
-  }
+  React.render(React.createElement(AppView, {
```

Comment:
I'd be happy to go through and extract the parameters into local variables, to make the expression of these props as compact as possible.  That said, I think some of that will be mitigated as we get further into this process and break up page.html.ejs into smaller pieces that need fewer parameters.